tests/unit/utils/__init__.py

```diff
@@ -387,3 +388,17 @@ def replace_config(n, **kwargs):
         config=n.config.replace(**kwargs),
         unrendered_config=dict_replace(n.unrendered_config, **kwargs),
     )
+
+
+def make_manifest(nodes=[], sources=[], macros=[], docs=[]) -> Manifest:
```

Comment:
I think this is duplicating https://github.com/dbt-labs/dbt-core/blob/84456f50f687b352072f64867247d21bef2647ff/tests/unit/utils/manifest.py#L986 here, thoughts on how can we make that one easier to use/more visible?